components/eam/src/physics/crm/crm_physics.F90

```diff
@@ -1330,15 +1328,15 @@ subroutine crm_physics_tend(ztodt, state, tend, ptend, pbuf2d, cam_in, cam_out,
          ! CRM momentum tendencies
          !------------------------------------------------------------------------------------------
 
-#if defined( MMF_USE_ESMT )
-         ptend(c)%lu = .TRUE.
-         ptend(c)%lv = .TRUE.
-         do i = 1,ncol
-            icrm = ncol_sum + i
-            ptend(c)%u(i,1:pver)  = crm_output%u_tend_esmt(icrm,1:pver)
-            ptend(c)%v(i,1:pver)  = crm_output%v_tend_esmt(icrm,1:pver)
-         end do
-#else /* MMF_USE_ESMT not defined */  
+         if (use_MMF_ESMT) then
```

Comment:
1. No. The ESMT scalars are always pointing in the zonal and meridional directions. They don't really exist with a flexible orientation like the wind of a 2D CRM does.

2. Correct. There can only be one set of momentum tendencies fed back to the GCM. Originally I made it so that they could run side-by-side in a 3D case. The idea there was to have instantaneous tendencies from both ESMT and the fully resolved CRM wind to make a really detailed comparison with a single run. But that was only for development purposes.